File: app/src/main/java/com/botijasoftware/utils/AtlasWrapper.java
package com.botijasoftware.utils;


//simple class that allows the use of a texture as a 1x1 atlas 
public class AtlasWrapper extends Atlas {
	public AtlasWrapper(Texture texture) {
		super(texture);
		mNumTextures = 1;
		mCoords = new TextureCoords[1];
		mCoords[0] = texture.getTextureCoords();
	}
}

File: app/src/main/java/com/botijasoftware/utils/AtlasWrapper.kt
package com.botijasoftware.utils

import java.util.ArrayList


//simple class that allows the use of a texture as a 1x1 atlas
class AtlasWrapper(texture: Texture) : Atlas(texture) {
    init {
        textureCount = 1
        mCoords = Array<TextureCoords>(1, {texture.textureCoords})
        //mCoords[0] = texture.textureCoords
    }
}